.travis.yml

```diff
@@ -68,6 +68,9 @@ jobs:
       script: ./ciRunSbt.sh clean ui/slow:test
     - name: "Management tests"
       script: ./ciRunSbt.sh clean management/it:test
+    - name: "Scala 2.11-2.12 compile"
```

Comment:
WDYT about changing name of this task to `Cross scala compile`. Let's keep information about all supported versions in one place, in sbt...

Reply:
:+1:

Reply:
fixed